server/Makefile

```diff
@@ -141,8 +141,8 @@ PLUGIN_PACKAGES += mattermost-plugin-github-v2.4.0
 PLUGIN_PACKAGES += mattermost-plugin-gitlab-v1.10.0
 PLUGIN_PACKAGES += mattermost-plugin-jira-v4.2.1
 # We need to prepackage both versions of playbooks and install the correct one based on the server license. See MM-60025.
-PLUGIN_PACKAGES += mattermost-plugin-playbooks-v1.40.0
-PLUGIN_PACKAGES += mattermost-plugin-playbooks-v2.1.1
+PLUGIN_PACKAGES += mattermost-plugin-playbooks-v1.41.0
```

Comment:
Do we need to bump v1 as well?

Reply:
If there's anything new with v1 I guess we should.

Reply:
Yes. v1 is still supported due to the 9.11 ESR.